pytest_pootle/search.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from itertools import groupby
+
+from django.conf import settings
+from django.core.urlresolvers import resolve
+from django.db.models import Max
+
+from pootle_misc.checks import get_category_id
+
+from pootle_store.models import Unit
+from pootle_store.unit.filters import UnitTextSearch, UnitSearchFilter
+from pootle_store.views import (
+    ALLOWED_SORTS, SIMPLY_SORTED, _path_units_with_meta)
+from virtualfolder.helpers import extract_vfolder_from_path
+from virtualfolder.models import VirtualFolderTreeItem
+
+
+def calculate_search_results(kwargs, user):
+    initial = kwargs["initial"]
+    pootle_path = kwargs["pootle_path"]
+
+    category = kwargs.get("category")
+    checks = kwargs.get("checks")
+    limit = kwargs.get("count", 9)
+    modified_since = kwargs.get("modified_since")
+    search = kwargs.get("search")
+    sfields = kwargs.get("sfields")
+    soptions = kwargs.get("soptions", [])
+    sort = kwargs.get("sort", None)
+    uids = kwargs.get("uids")
+    unit_filter = kwargs.get("filter")
+
+    vfolder = None
+    if 'virtualfolder' in settings.INSTALLED_APPS:
+        vfolder, pootle_path = extract_vfolder_from_path(
+            pootle_path,
+            vfti=VirtualFolderTreeItem.objects.select_related(
+                "directory", "vfolder"))
+    qs = (
+        Unit.objects.get_translatable(user=user, **resolve(pootle_path).kwargs)
+                    .filter(store__pootle_path__startswith=pootle_path)
+                    .order_by("store", "index"))
```

Comment:
hmm - not sure - this is/was the default ordering.

if/when we add order_by further down it removes this